docs/headless-cms/03-03-markdown-documents.md

```diff
@@ -29,7 +29,12 @@ type: {Document type}
 ---
 ```
 
-- The `title` metadata defines the title of your document.
+- The `title` metadata defines the title of your document. If you want the title or its part to render inside quotation marks, place it between additional, single quotation marks or use a backslash (**/**) before the quoted word. See the following examples:
+
+  ✅ `title: '"Transport is closing" error'`<br>
+  ✅ `title: \"Transport is closing" error`<br>
```

Comment:
it should be `title: \"Transport is closing\" error`

http://docs.octoprint.org/en/master/configuration/yaml.html
```
In double quoted strings if you need to include a literal double quote in your string you can escape it by prefixing it with a backslash \ (which you can in turn escape by itself). In single quoted strings the single quote character can be escaped by prefixing it with another single quote, basically doubling it. Backslashes in single quoted strings do not need to be escaped.
```